app.rb

```diff
@@ -12,6 +12,7 @@
 require 'everypolitician/popolo'
 
 require_relative './lib/popolo_helper'
+require_relative './helpers/html'
```

Comment:
Shouldn't we be consistent where the helpers live? If we're making a new 'helpers' directory for them, then we should probably move the `popolo_helper` there too.